tests/Makefile

```diff
@@ -4,6 +4,7 @@ export CFLAGS+=-g -O0 -Wall -D_GNU_SOURCE
 DISTRO = $(shell ./os_detect)
 
 SUBDIRS := \
+	exclude_filter \
```

Comment:
"exclude" is the name of the filter action.  Similarly "sessionID" is the name, while filter is the action.